src/pages/dashboard/DashboardPage.tsx

```diff
@@ -115,30 +132,42 @@ function DashboardPage() {
       resizeHandle={renderResizeHandle()}
       containerPadding={[0, 0]}
     >
-      <div key="queueProcessor">
-        <QueueProcessor />
-      </div>
+      {!hideQueueProcessor && (
+        <div key="queueProcessor">
+          <QueueProcessor />
+        </div>
+      )}
       <div key="importBreakdown">
         <UnrecognizedFiles />
       </div>
-      <div key="recentlyImported">
-        <RecentlyImported />
-      </div>
-      <div key="collectionBreakdown">
-        <CollectionStats />
-      </div>
+      {!hideRecentlyImported && (
+        <div key="recentlyImported">
+          <RecentlyImported />
+        </div>
+      )}
+      {!hideCollectionBreakdown && (
+        <div key="collectionBreakdown">
+          <CollectionStats />
+        </div>
+      )}
       <div key="collectionTypeBreakdown">
         <MediaType />
       </div>
-      <div key="importFolders">
-        <ImportFolders />
-      </div>
-      <div key="shokoNews">
-        <ShokoNews />
-      </div>
-      <div key="continueWatching">
-        <ContinueWatching />
-      </div>
+      {!hideImportFolders && (
+        <div key="importFolders">
+          <ImportFolders />
+        </div>
+      )}
+      {!hideShokoNews && (
+        <div key="shokoNews">
+          <ShokoNews />
+        </div>
+      )}
+      {!combineContinueWatching && (
+        <div key="continueWatching">
+          <ContinueWatching />
+        </div>
+      )}
       <div key="nextUp">
         <NextUp />
```

Comment:
Same as above, but also toggling hiding the next up and continue watching sections separately, and not just combining them.